Worker.Dockerfile

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: 2024 HH Partners
+# SPDX-FileCopyrightText: 2024 Double Open Oy
```

Comment:
The changes in this file are unrelated to the commit topic and should be split out to a separate commit.

Reply:
So do you think copyright change should always be a separate commit, and not done when the file is modified?

Reply:
Uhm, not necessarily, although it's probably a good practice. But it seemed to me the file is not modified (besides adding a trailing newline) in that commit. Let me re-check...

Reply:
Is separated it.

Reply:
This is the commit that should have the added newline in the end.